src/contracts/Model/Suggestion/ContentSuggestion.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Search\Model\Suggestion;
+
+final class ContentSuggestion extends Suggestion
+{
+    private int $contentId;
+
+    private ?int $locationId;
+
+    private string $contentTypeIdentifier;
+
+    /**
+     * @param array<\Ibexa\Contracts\Search\Model\Suggestion\ParentLocation> $parentLocations
+     */
+    public function __construct(
+        float $score,
+        string $contentTypeIdentifier,
+        string $name,
+        int $contentId,
+        ?int $locationId = null,
+        string $pathString = '',
+        array $parentLocations = []
+    ) {
+        parent::__construct($score, $name, $pathString, $parentLocations);
+        $this->contentId = $contentId;
+        $this->contentTypeIdentifier = $contentTypeIdentifier;
+        $this->locationId = $locationId;
+    }
+
+    public function getContentId(): int
+    {
+        return $this->contentId;
+    }
+
+    public function getLocationId(): ?int
+    {
+        return $this->locationId;
+    }
+
+    public function getContentTypeIdentifier(): string
+    {
+        return $this->contentTypeIdentifier;
```

Comment:
Why not to include proper API objects? In many cases having only identifier is not enough, especially if it's passed directly to templates where you don't really have an option to fetch `ContentType` object.